src/core/global.cpp

```diff
@@ -370,6 +375,12 @@ void Initialize( int& argc, char**& argv )
     const long seed = (secs<<16) | (rank & 0xFFFF);
     ::generator.seed( seed );
     srand( seed );
+
+#ifndef EL_RELEASE
+        char sbuf[50];
+        sprintf(sbuf,"El-Proc%03d.log",rank);
+        LogFileOpen(sbuf);
+#endif
```

Comment:
Here I prefer to add leading '0' for the index of process. The alternative way will be
```c++
#include< iomanip>

DEBUG_ONLY(
    ostringstream fileOS;
    fileOS << "El-Proc" << std::setfill('0') << std::setw(3) << rank << ".log";
    OpenLog( fileOS.str().c_str() );
)
```